### PR TITLE
fix(p6-translation): fixes the alignment of the component with the other fields of the form

### LINKED ISSUE #21: the alignment of p6-translation is incorrect with the other form fields

### PR DESCRIPTION
**Pull request by vincentv**
fix(p6-translation): fixes the alignment of the component with the other fields of the form

Closes #21

### CODE REVIEW

File: packages/core/src/components/molecules/p6-translation/p6-translation.stories.ts [hunk 1]
```diff
@@ -53,11 +53,19 @@ export const Form = makeStory<{
...
+          slot: 'label',
+        }),
+        getElement('p6-input', '', {
+          name: 'test',
+        }),
+      ]),
       getStoryField('Label', {
         name: 'field',
-        value: { en: 'English', fr: 'Français' },
+        value: { en: 'English2', fr: 'Français' },
```
Comment (jswale) [inline packages/core/src/components/molecules/p6-translation/p6-translation.stories.ts:67]: why a 2 at the end ?

**Review by jswale — APPROVED** [review]

**Comment by vincentv** [comment]
image

with the responsive rules active : 
image

**Comment by vuongc** [comment]
Why the text from the textarea seems to be more compressed than the select's one ?

**Review by jswale — APPROVED** [review]

**Comment by vincentv** [comment]
after the modifications reported by @vuongc 

image

with the responsive rules active :
image

**Review by vuongc — APPROVED** [review]

**Pull request merged**